.do/remap-api-url.js

```diff
@@ -0,0 +1,13 @@
+const replace = require("replace-in-file")
+
+try {
+  console.log(
```

Comment:
Do we want to log the output of the replacement?

Reply:
![image](https://user-images.githubusercontent.com/28274198/96388532-7e23fc00-1177-11eb-8bee-b5e7c667f7f2.png)

Reply:
Gives a nice log output to see if its was succesful